app/views/assignments/_admin_subs.html.erb

```diff
@@ -274,6 +278,10 @@
         widgets: ['uitheme']
       });
       $(".no-resort").on("mousedown", function(e) { e.stopPropagation(); });
+      $("#show_username").click(function(){
```

Comment:
Let's move this code into a function in app/assets/javascripts/application.js,
```js
function enableShowUsernames(elts) {
  $(elts).click(....);
}
```
and then it's callable as `enableShowUsername("#show_username")` from this file.  That way, we can enable this in more places more easily.

Reply:
I have moved it into app/assets/javascripts/application.js 
Is there a specific way Rails handles JS files, as it complains that it cannot find enableShowUsername in  app/views/assignments/_admin_subs.html.erb